src/components/NotFound/NotFound.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import EagleAnimation from './EagleAnimation'
+import { theme, breakpoint, Button } from '@aragon/ui'
+const medium = css => breakpoint('medium', css)
+
+class NotFound extends React.Component {
+  state = { details: false }
+  clickMoreDetails = () => {
+    const details = this.state.details
+    this.setState({ details: !details })
+  }
+  render() {
+    const {
+      title,
+      issue,
+      reload,
+      detailsTitle,
+      detailsContent,
+      children,
+    } = this.props
+    const { details } = this.state
+    return (
+      <div>
+        <EagleAnimation />
+        <Card>
+          <h1>{title}</h1>
+          {children}
+          {(detailsContent || detailsTitle) && (
+            <div>
+              <DetailsButton onClick={this.clickMoreDetails}>
+                {details ? 'Hide details...' : 'More details...'}
+              </DetailsButton>
+              {details && (
+                <DetailsContainer>
+                  {detailsTitle && <h2>{detailsTitle}</h2>}
+                  {detailsContent}
+                </DetailsContainer>
+              )}
+            </div>
+          )}
+          {(issue || reload) && (
+            <ButtonBox>
+              {issue && (
+                <IssueLink>
+                  <Button.Anchor
+                    mode="text"
+                    href="https://github.com/aragon/aragon/issues"
+                    target="_blank"
+                  >
+                    Tell us what went wrong
+                  </Button.Anchor>
+                </IssueLink>
+              )}
+              {reload && (
+                <Button mode="strong" onClick={() => location.reload()}>
```

Comment:
We usually declare class methods for event handlers instead of inline functions, naming them `handleSomething()`. In that case, it could be `handleReloadClick()`.